docs_2.0/manual_EN/4.ngql-guide/7.general-query-statements/2.match.md

```diff
@@ -70,12 +80,74 @@ Search for vertices with properties matching a certain pattern:
     Got 2 rows (time spent 2407/3244 us)
     ```
 
-## MATCH on edge types
+* Example 3: Search for the player whose name is "Tim", return the player's relationships (edges) with others, as well as the names of these people. The `r` is a user-defined variable that represents an edge. `type(e)` to refer to the edge type.
+
+    ```nGQL
+    nebula> MATCH (v1:player{name: "Tim"}) -[e]-> (v2) \
+         -> RETURN type(r) AS Relationship, v2.name AS Name;
+    +--------------+-------------------+
+    | Relationship | Name              |
+    +--------------+-------------------+
+    | follow       | Tony Parker       |
+    +--------------+-------------------+
+    | follow       | LaMarcus Aldridge |
+    +--------------+-------------------+
+    | serve        | Warriors          |
+    +--------------+-------------------+
+    Got 3 rows (time spent 4844/6660 us)
+    ```
 
-### One-hop MATCH
+* Example 3: Search for the team that "Tim" serves, return the team name, the start year, and the end year.
 
-[TODO]
+    ```nGQL
+    nebula> MATCH (v1:player{name: "Tim"}) -[e:serve]-> (v2) \
+         -> RETURN v2.name AS Team_Name, e.start_year AS Start_Year, e.end_year AS End_Year;
+    +-----------+------------+----------+
+    | Team_Name | Start_Year | End_Year |
+    +-----------+------------+----------+
+    | Warriors  | 1997       | 2016     |
+    +-----------+------------+----------+
+    Got 1 rows (time spent 2768/3412 us)
+    ```
 
-## Two-hop MATCH
+* Example 4: Search for the player whose name is "Tim", find the people "Tim" follows with age above 35 and the teams they serve, and return the names of the people and teams.
 
-[TODO]
+    ```nGQL
+    nebula> MATCH (v1:player) -[:follow]-> (v2) -[:serve]-> (v3) \
+         -> WHERE v1.name == "Tim" AND v2.age > 35 \
+         -> RETURN v1.name AS Player, v2.name AS Friend, v3.name AS Friends_Team;
+    +--------+-------------+--------------+
+    | Player | Friend      | Friends_Team |
+    +--------+-------------+--------------+
+    | Tim    | Tony Parker | Nuggets      |
+    +--------+-------------+--------------+
+    Got 1 rows (time spent 3623/4220 us)
+    ```
+
+### Examples for matching on ID
+
+* Example 1: Search for the vertex with VID "player100" and return all its information.
+
+    ```nGQL
+    nebula> MATCH (v) WHERE id(v) == 'player101' RETURN v;
+    +---------------------------------------------------+
+    | v                                                 |
+    +---------------------------------------------------+
+    | (player101) player.name:Tony Parker,player.age:36 |
+    +---------------------------------------------------+
+    Got 1 rows (time spent 1710/2406 us)
+    ```
+
```

Comment:
```
Compatible issue: in nGQL, `id()` will return a vertex's VID, which can be either a int64 or fixed_string. In OpenCypher `id()` returns an Interger.
```

Reply:
For now nGQL returns a string.
Note added.